src/app/screens/Accounts.tsx

```diff
@@ -0,0 +1,126 @@
+  }
+
+  async function selectAccount(accountId: string) {
+    auth.setAccountId(accountId);
+    await utils.call("selectAccount", {
+      id: accountId,
+    });
+    auth.fetchAccountInfo(accountId);
+  }
+
+  async function removeAccount(id: string, name: string) {
+    if (window.confirm(`Are you sure you want to delete account: ${name}?`)) {
+      let nextAccountId;
+      let accountIds = Object.keys(accounts);
+      if (auth.account?.id === id && accountIds.length > 1) {
+        nextAccountId = accountIds.filter((accountId) => accountId !== id)[0];
+      }
+
+      await Promise.all([
+        utils.call("deleteAccount", { id }),
+        removeAccountFromCache(id), // Clear cached account info (alias/balance).
+      ]);
+      accountIds = accountIds.filter((accountId) => accountId !== id);
+
+      if (accountIds.length > 0) {
+        getAccounts();
+        if (nextAccountId) selectAccount(nextAccountId);
+      } else {
+        window.close();
+      }
+    }
+  }
+
+  useEffect(() => {
+    getAccounts();
+  }, []);
+
+  return (
+    <Container>
+      <h2 className="mt-12 mb-6 text-2xl font-bold dark:text-white">
+        Accounts
+      </h2>
+      <div className="shadow overflow-hidden border-b border-gray-200 sm:rounded-lg">
+        <table className="min-w-full divide-y divide-gray-200">
+          <thead className="bg-gray-50">
+            <tr>
+              <th
+                scope="col"
+                className="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider"
+              >
+                Name
+              </th>
+              <th
+                scope="col"
+                className="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider"
+              >
+                <span className="sr-only">Badge</span>
+              </th>
+              <th scope="col" className="relative px-6 py-3">
+                <span className="sr-only">Delete</span>
+              </th>
+            </tr>
+          </thead>
+          <tbody className="bg-white divide-y divide-gray-200">
+            {Object.keys(accounts).map((accountId) => {
+              const account = accounts[accountId];
+              return (
+                <tr key={accountId}>
+                  <td className="px-6 py-4 whitespace-nowrap">
+                    <div className="flex items-center">
+                      <WalletIcon className="w-8 h-8 -ml-1 text-gray-500" />
+                      <div className="ml-4">
+                        <div className="text-sm font-medium text-gray-900">
+                          {account.name}
+                        </div>
+                        <div className="text-sm text-gray-500">
+                          {account.connector}
+                        </div>
+                      </div>
+                    </div>
+                  </td>
+                  <td className="px-6 py-4 whitespace-nowrap">
+                    <Badge
```

Comment:
How about we remove this centered column?
I don't really think that connector badge provides much value (initially I put the connector there because in the design there was a badge and I did not have anything else 🙈😱)